demo/create_result_gif.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import os
+import os.path as osp
+
+import imageio
```

Comment:
Is this a new dependency? can we use API in cv2?

Reply:
I did not find the cv2 to support gif creation, But I found that PIL supports, and the gif generated by PIL does not work well even if the `quality` parameter is added, the effect is the same

`
images=[Image.fromarray(frame) for frame in frames]
images[0].save(gif_name,save_all=True,append_images=images, duration=duration,loop=0,quality=95)
`

![image](https://user-images.githubusercontent.com/17425982/107030620-deed4800-67eb-11eb-9be0-0e1c4442e72d.png)